examples/llamaindex_examples/fastapi_streaming_response/stream.py

```
@@ -0,0 +1,268 @@
+    tracer = Tracer(
+        project_name="test_stream", # Match UI project name
+        dataset_name="fleety_chat_after_fix", # Unique dataset name
+        tracer_type="Agentic"
+    )
+    init_tracing(catalyst=catalyst, tracer=tracer)
+    return tracer
+# FastAPI app
+app = FastAPI(title="ReAct Agent API")
+# Initialize tracing
+tracer = initialize_tracing()
+from llama_index.core.llms import ChatMessage
```

Ruff (0.8.2)

Redefinition of unused `ChatMessage` from line 10

Remove definition: `ChatMessage`

(F811)